example/tokenization/readme.md

```diff
@@ -0,0 +1,4 @@
+How to run subscription example:
+
+1. Install dependencies: `gem install veritrans`
+2. Run the file: `ruby gopay_tokenization_example.rb` 
```

Comment:
I ran the instruction, I got this on terminal
```
gitpod /workspace/veritrans-ruby/example/tokenization $ ruby gopay_tokenization_example.rb 
...

Traceback (most recent call last):
        4: from gopay_tokenization_example.rb:87:in `<main>'
        3: from /home/gitpod/.rvm/rubies/ruby-2.7.4/lib/ruby/2.7.0/forwardable.rb:235:in `charge'
        2: from /home/gitpod/.rvm/gems/ruby-2.7.4/gems/veritrans-2.4.0/lib/veritrans/api.rb:45:in `charge'
        1: from /home/gitpod/.rvm/gems/ruby-2.7.4/gems/veritrans-2.4.0/lib/veritrans/client.rb:41:in `request_with_logging'
/home/gitpod/.rvm/gems/ruby-2.7.4/gems/veritrans-2.4.0/lib/veritrans/client.rb:128:in `make_request': Midtrans API is returning API error. HTTP status code: 400 (MidtransError)
```
Why don't you implement a proper try-catch like all other example we have? do this to the other example files too.

Reply:
for this gopay example, error because i doing charge using inactive/disable account_id
will add comment # this is only sample charge and will do implement handling error 🙏